src/components/Picker/Picker.tsx

```diff
@@ -43,14 +57,18 @@ export class Picker extends React.Component<PickerProps> {
       onNext,
       prevDisabled,
       nextDisabled,
+      prevAriaLabel,
+      nextAriaLabel,
       className,
+      ...rest
     } = this.props;
 
     return (
       <div
         data-hook={this.props['data-hook']}
         className={st(classes.root, {}, className)}
         {...this.getDataAttributes()}
+        {...rest}
```

Comment:
Do we need to pass all the `rest` to the `div`?

Reply:
i just thought that it will allow consumers add props that they need, but thinking about it, typescript will not allow it anyway.

Reply:
that's cause the IconButton extends ButtonNext, so the ...rest is for the rest of ButtonNext props.
In this case it's not needed

Reply:
the aria-label prop, how's holding it ?